vpn/vpn.proto

```diff
@@ -1,5 +1,6 @@
 syntax = "proto3";
 option go_package = "github.com/coder/coder/v2/vpn";
+option csharp_namespace = "Coder.Desktop.Rpc.Proto";
```

Comment:
Coder Desktop is the entire desktop application, and Coder VPN is the feature. So, I think we should qualify the namespace to "VPN" somehow, since this protocol really is specific to Coder VPN.

Reply:
The whole solution is named `Coder.Desktop`. I could rename the solution to `Coder.Vpn` but it would make less sense because then the UI project would be named `Coder.Vpn.Desktop` (or something). We want to keep all the code in a single solution so we don't need to mess with packaging the library packages for consumption in a different solution.

There's also no VPN-specific components in the C# code unlike with the Mac version, just a service manager and UI, which makes me inclined to keep it named as `Coder.Desktop`.

Reply:
I can move it to `Coder.Desktop.Vpn.Rpc` or `Coder.Desktop.VpnRpc` perhaps?

Reply:
I think the whole solution should still be `Coder.Desktop`, and this could be `Coder.Desktop.Vpn.Rpc` (or even just `Coder.Desktop.Vpn` if you don't think we need the "RPC" qualifier).  Not sure what feels more natural in C#.

Reply:
Yeah we could do `Coder.Desktop.Vpn`. I might rename some of my types to e.g. `RpcSpeaker` though

Reply:
The protocol will still be `Coder.Desktop.Vpn.Proto` since it's in it's own project